gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
```

Comment:
JFYI: this also allows to run build on JDK17+, so later on we can modify CI recipe to do that

Reply:
yes, but that is not at all representative of the "typical" JVM this code is expected to run on?

Reply:
Yeah, all of the clients are on JDK8 or 11 I assume, that's why I said it paves the way to build/test on JDK17 in future, but no obligations to implement it.